### PR TITLE
chore: remove in-page rendering

### PR DESCRIPTION
**Pull request by 2color**
## Description

This is a small PR to narrow functionality scope by removing in-page rendering. 

## Change checklist

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation if necessary (this includes comments as well)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: src/components/CidRenderer.tsx [hunk 1]
```diff
@@ -22,38 +22,6 @@ import React, { useState } from 'react'
  *
  */
 
-export function ContentRender ({ blob, contentType, text, path, isLoading }): JSX.Element {
```
Comment (SgtPooki) [inline src/components/CidRenderer.tsx:25]: i'm so sad to see this component go away, but it needs to 😢

File: src/components/CidRenderer.tsx [hunk 2]
```diff
@@ -100,47 +62,14 @@ export default function CidRenderer ({ requestPath }: { requestPath: string }):
...
-
   return (
     <div>
       <ValidationMessage pathNamespacePrefix={pathNamespacePrefix} cid={cid} requestPath={requestPath}>
-        <button id="load-in-page" onClick={() => { void makeRequest() }} className='button-reset pv3 tc bn bg-animate bg-black-80 hover-bg-aqua white pointer w-100'>Load in-page</button>
 
-        <a className="pt3 db" href={swPath} target="_blank">
-          <button id="load-directly" className='button-reset pv3 tc bn bg-animate bg-black-80 hover-bg-aqua white pointer w-100'>Load directly / download</button>
+        <a className="db" href={swPath} target="_blank">
+          <button id="load-directly" className='button-reset pv3 tc bn bg-animate bg-black-80 hover-bg-aqua white pointer w-100'>Load CID</button>
```
Comment (SgtPooki) [inline src/components/CidRenderer.tsx]:
```suggestion
          <button id="load-directly" className='button-reset pv3 tc bn bg-animate bg-black-80 hover-bg-aqua white pointer w-100'>Load content</button>
```

**Review by SgtPooki — APPROVED** [review]
Minor nit, but approved

File: src/components/CidRenderer.tsx (hunk 1 above)
Comment (2color) [reply src/components/CidRenderer.tsx:25]: I can imagine!

**Pull request merged**